### PR TITLE
Added home button

### LINKED ISSUE #58: Where is the "home" button
I'm missing the "Home" button which brings me to the welcome page of a ZIM. It seems to be absent from the design too! @schoenbaechler What that on purpose?

### PR DESCRIPTION
**Pull request by sgourdas**
This aims to close #58 

Added the home button beside the search bar, that works using the pre existing logic tied to `alt+home`

### CODE REVIEW

**Comment by kelson42** [comment]
@sgourdas For me it's OK, on the usabilty side, but please make the logo a bit bigger, compare with the other logo, I guess a few pixels are missing

**Comment by sgourdas** [comment]
> There is also a problem with the color of the line (same for the random button): you have but black, but if you look to the icon on the right, they are different, please fix.

@kelson42 sorry but this is not clear to me. Do you mean that the left icons (home and random) are bolder than the right ones?

**Comment by sgourdas** [comment]
Checking the `.svg` of each file respectively they all use `#000` for color which is black. For me, the only visible difference is the boldness. Should I check something different?

**Comment by kelson42** [comment]
@sgourdas It's not the same coloring, please see 
image

**Comment by sgourdas** [comment]
@kelson42 I got what you mean. I did some improvements to get to the look intended. 

Regarding the **random** icon, I cannot get it to be thinner than this. An alternative I have ready, if you prefer, would be using this one below:
image

**Comment by kelson42** [comment]
@sgourdas At a first sight, it looks indeed better but Iwill review in detaila coloring, boldness and sizing. Hope all good.

**Comment by kelson42** [comment]
@veloman-yunkan Can you please validate the code?

**Comment by kelson42** [comment]
@sgourdas Like other buttons related to the content, the button should be deactivated if we are in the library

**Comment by sgourdas** [comment]
@kelson42 added the disabling functionality and reduced some instance calls also in the meantime.

**Review by kelson42 — APPROVED** [review]
LGTM from a user perspective, @veloman-yunkan ready for code review

**Review by veloman-yunkan — CHANGES_REQUESTED** [review]
LGTM, but please split your changes into two commits:

1. Code cleanup
2. Implementation of the new feature

**Review by veloman-yunkan — COMMENTED** [review]
The title of the second commit "Code cleanup and random.svg color alignment" suggests that two unrelated changes are combined into one.

File: src/tabbar.cpp [hunk 1]
```diff
@@ -66,6 +62,12 @@ TabBar::TabBar(QWidget *parent) :
             this, SLOT(onTabMoved(int,int)), Qt::DirectConnection);
 }
 
+void TabBar::openHomePage() {
+    auto current = this->currentWebView();
+    QUITIFNULL(current);
+    current->setUrl("zim://" + current->zimId() + ".zim/");
+}
```
Comment (veloman-yunkan) [inline src/tabbar.cpp:69]: This is rather a refactoring change. Maybe it better belong to the code cleanup commit.

**Comment by sgourdas** [comment]
> The title of the second commit "Code cleanup and random.svg color alignment" suggests that two unrelated changes are combined into one.

Should i split that also in 2? `random.svg` is being changed in that to allow for better consistency with the rest toolbar icons.

**Review by veloman-yunkan — COMMENTED** [review]
I'd prefer if code cleanup precedes the enhancement (since in general that is how enhancements should be done - while studying the code, you discover ways in which it could be improved, so that the enhancement can benefit from those changes too).

**Comment by veloman-yunkan** [comment]
> > The title of the second commit "Code cleanup and random.svg color alignment" suggests that two unrelated changes are combined into one.
> 
> Should i split that also in 2? `random.svg` is being changed in that to allow for better consistency with the rest toolbar icons.

Yes. Changes should be atomic. I would even recommend having two separate commits (extracting a piece of code into a named function; shortening lines with the help of auxiliary local variables) for the refactoring/cleanup part of this PR

File: src/tabbar.cpp (hunk 1 above)
Comment (sgourdas) [reply src/tabbar.cpp:69]: Agree

Comment (sgourdas) [reply src/tabbar.cpp:69]: BTW, do you think this change is beneficial since `openHomePage` is only called in one place?

Comment (veloman-yunkan) [reply src/tabbar.cpp:69]: Of course, it is. Now now that place is less cluttered.

**Comment by sgourdas** [comment]
@veloman-yunkan is this ok?

File: src/kiwixapp.cpp [hunk 2]
```diff
@@ -527,6 +527,7 @@ void KiwixApp::handleItemsState(TabType tabType)
     KiwixApp::instance()->getAction(KiwixApp::ZoomOutAction)->setDisabled(libraryOrSettingsTab);
     KiwixApp::instance()->getAction(KiwixApp::ZoomResetAction)->setDisabled(libraryOrSettingsTab);
     KiwixApp::instance()->getAction(KiwixApp::RandomArticleAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::OpenHomePageAction)->setDisabled(libraryOrSettingsTab);
```
Comment (veloman-yunkan) [inline src/kiwixapp.cpp:531]: This change won't compile - there is no variable `app` yet. Please fix it by moving the last (code cleanup) commit to the beginning of the PR.

Comment (sgourdas) [reply src/kiwixapp.cpp:531]: Fixed, sorry for this.

**Review by veloman-yunkan — APPROVED** [review]

**Pull request merged**